tests/test_coordinated_workers/test_coordinator.py

```diff
@@ -281,3 +284,46 @@ def test_tracing_receivers_urls(
             "otlp_http": "5.6.7.8:4318",
             "otlp_grpc": "5.6.7.8:4317",
         }
+
+
+def test_invalid_databag_content(coordinator_charm: ops.CharmBase):
+    requires_relations = {
+        endpoint: testing.Relation(endpoint=endpoint, interface=interface["interface"])
+        for endpoint, interface in {
+            "my-certificates": {"interface": "certificates"},
+            "my-logging": {"interface": "loki_push_api"},
+            "my-charm-tracing": {"interface": "tracing"},
+            "my-workload-tracing": {"interface": "tracing"},
+        }.items()
+    }
+    requires_relations["cluster_worker0"] = testing.Relation(
+        "my-logging",
+        remote_app_name="worker0",
+        remote_app_data=ClusterRequirerAppData(role="read").dump(),
+    )
+    requires_relations["cluster_worker2"] = testing.Relation(
+        "my-logging",
+        remote_app_name="worker2",
+    )
+
+    provides_relations = {
+        endpoint: testing.Relation(endpoint=endpoint, interface=interface["interface"])
+        for endpoint, interface in {
+            "my-dashboards": {"interface": "grafana_dashboard"},
+            "my-metrics": {"interface": "prometheus_scrape"},
+        }.items()
+    }
+
+    invalid_databag_state = testing.State(
+        containers={
+            testing.Container("nginx", can_connect=True),
+            testing.Container("nginx-prometheus-exporter", can_connect=True),
+        },
+        relations=list(requires_relations.values()) + list(provides_relations.values()),
+    )
+
+    ctx = testing.Context(coordinator_charm, meta=coordinator_charm.META)
+    with ctx(ctx.on.start(), invalid_databag_state) as manager:
```

Comment:
to clarify that we're not testing the 'start' event specifically, but any, you could parametrize and add a couple more. For example:

```python
@pytest.mark.parametrize("event",
(
scenario.CharmEvents.update_status(),
scenario.CharmEvents.start(),
scenario.CharmEvents.install(),
) 
)
```